src/adjoint/masterRoutines.F90

```diff
@@ -1,7 +1,9 @@
 module masterRoutines
 contains
-  subroutine master(useSpatial, famLists, funcValues, forces, &
-       bcDataNames, bcDataValues, bcDataFamLists)
+  subroutine master(useSpatial,&
+                  famLists, funcValues,&
+                  forces, &
+                  bcDataNames, bcDataValues, bcDataFamLists)
```

Comment:
again, this looks like a formatting change, correct? Is there a specific styling you were going for or is this just random? I would suggest not making modifications like this unless we fix entire modules' syntax at a time. Otherwise its just adding confusion and extra lines to the PR.

Reply:
This has been undone